src/main/java/edu/mbrooks/advancedjava/main/Collectable.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Program Name:    Collectable.java
+ * Author:          Michael Brooks
+ * Version:         Version 1.0
+ * Course:          Advanced Java Programming, Summer 2018
+ * Copyright:       (C) 2018 Michael A. Brooks
+ * Description:     Implementation of the collectable interface
+ */
+package edu.mbrooks.advancedjava.main;
+
+/**
+ *
+ * @author michaelbrooks
+ * This is a collection of something
```

Comment:
this is too vague.
why is this class needed? how does it differ from all the other collection classes.